sphinx-docs/installguide/install_linux.rst

```diff
@@ -0,0 +1,23 @@
+Linux Installation Guide
+===========================
+.. note:: You will need to make sure *sudo* is installed for both Debian and Ubuntu. These commands can then be used for both operating systems. 
+
```

Comment:
Maybe add another instruction to open the commandline terminal, in case they don't know where to type these instructions?